src/i18n/messages-es.po

```diff
@@ -0,0 +1,881 @@
+# Spanish translations for Spreed WebRTC.
+# Copyright (C) 2016 struktur AG
+# This file is distributed under the same license as the Spreed WebRTC
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Spreed WebRTC 1.0\n"
+"Report-Msgid-Bugs-To: simon@struktur.de\n"
+"POT-Creation-Date: 2016-07-02 14:57-0500\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Jhon Felipe Urrego Mejia <ingenierofelipeurrego@gmail.com>\n"
+"Language: es\n"
+"Language-Team: es <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.3.4\n"
+
+#, python-format
+msgid "and %s"
+msgstr "y %s"
+
+#, python-format
+msgid "and %d others"
+msgstr "y %d otros"
+
+msgid "User"
+msgstr "Usuario"
+
+msgid "Someone"
+msgstr "Alguien"
+
+msgid "Me"
+msgstr "Yo"
+
+msgid "Peer to peer chat active."
+msgstr "Chat abierto de igual a igual."
+
+msgid "Peer to peer chat is now off."
+msgstr "Chat abierto de igual a igual ahora está apagado."
+
+msgid " is now offline."
+msgstr " ahora está desconectado."
```

Comment:
I would say "está ahora desconectado." to keep consistency with "está ahora en línea."